scripts/scil_compute_maps_for_particle_filter_tracking.py

```diff
@@ -27,31 +27,31 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(
         description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
     p.add_argument(
```

Comment:
Maybe its just me, but I prefer when as much argument are on the same line (in this case, 'in_wm')